### PR TITLE
Ltfoa solarenergie daecher

### PR DESCRIPTION
**Pull request by AFoletti**

### CODE REVIEW

**Comment by AFoletti** [comment]
rebased my branch and created a WIP pr. Tooltip is in a very early stage

File: chsdi/models/vector/uvek.py [hunk 1]
```diff
@@ -2466,3 +2467,75 @@ class ChmobilMountainbikeland (Base, Vector):
...
+    date_start = Column('date_start', Date)
+    sis_array = Column('array_values', postgresql.ARRAY(Numeric))
+    the_geom = Column(Geometry(geometry_type='GEOMETRY',
+                               dimension=2, srid=21781))
+
+register('ch.meteoschweiz.globalstrahlung-monatlich', globalstrahlung_sis)
+
+
+class globalstrahlung_sisdir (Base, Vector):
+    __tablename__ = 'meteoschweiz_globalstrahlung_sis'
```
Comment (loicgasser) [inline chsdi/models/vector/uvek.py]: same table name and schema twice?

Comment (AFoletti) [inline chsdi/models/vector/uvek.py]: yes, sorry. I'll post a temporary fix but the two views will be merged soon

**Comment by cype** [comment]
Inputs for the normal ObjectInfo:
- please round AREA to integer
- class is a value, please translate to multi-lingual-text like in translate.js
- link directly to the UI sonnendach

**Comment by AFoletti** [comment]
I addressed the issues mentioned by @cype 
I just need the sonnendach link now. @gjn ?

**Comment by AFoletti** [comment]
jenkins test

**Pull request merged**